examples/05-Retrieval-Model.ipynb

```diff
@@ -999,7 +999,9 @@
    "source": [
```

Comment:
<p>And here, please write the following instead:</p><p> </p><p> </p><p>We use&nbsp;<em>unique_rows_by_features</em>&nbsp;utility function to extract user features table and return it as a cuDF data frame. Note that for user features table, we use&nbsp;<code>USER</code>&nbsp;and&nbsp;<code>USER_ID</code>&nbsp;tags.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/967/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>